ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -3680,6 +3691,1087 @@ router bfd
+      redistribute isis level-1-2 rcf RCF_Address_Family_IPV6_ISIS()
+      redistribute ospfv3
+      redistribute ospfv3 match external include leaked
+      redistribute ospfv3 match nssa-external 1
+      redistribute static route-map RM-IPV6-STATIC-TO-BGP
+   !
+   address-family ipv6 multicast
+      bgp missing-policy direction in action permit
+      bgp missing-policy direction out action permit
+      bgp additional-paths receive
+      no neighbor FOOBAR activate
+      neighbor FOOBAR additional-paths receive
+      neighbor FOOBAR1 activate
+      neighbor aa::1 activate
+      neighbor aa::1 additional-paths receive
+      neighbor aa::1 route-map IPv6_MULTICAST_RM_IN in
+      neighbor aa::1 route-map IPv6_MULTICAST_RM_OUT out
+      network aa::1/126 route-map IPv6_MULTICAST_RM
+      redistribute connected route-map RM-address_family_ipv6_multicast-Connected
+      redistribute isis level-1 include leaked route-map RM-address_family_ipv6_multicast-ISIS
+      redistribute ospf route-map RM-address_family_ipv6_multicast-OSPF
+      redistribute ospfv3 route-map RM-address_family_ipv6_multicast-OSPFv3
+      redistribute ospfv3 match external route-map RM-address_family_ipv6_multicast-OSPFv3-External
+      redistribute ospfv3 match nssa-external 2 route-map RM-address_family_ipv6_multicast-OSPFv3-External
+      redistribute ospf match external route-map RM-address_family_ipv6_multicast-OSPF-External
+      redistribute ospf match nssa-external 2 route-map RM-address_family_ipv6_multicast-OSPF-External
+      redistribute static route-map RM-address_family_ipv6_multicast-Static
+   !
+   address-family ipv6 sr-te
+      neighbor SR-TE-PG-2 activate
+      neighbor SR-TE-PG-2 route-map RM-SR-TE-PEER-IN6 in
+      neighbor SR-TE-PG-2 route-map RM-SR-TE-PEER-OUT6 out
+      no neighbor SR-TE-PG-3 activate
+      neighbor 2001:db8::dead:beef:cafe activate
+      neighbor 2001:db8::dead:beef:cafe route-map RM-SR-TE-PEER-IN6 in
+      neighbor 2001:db8::dead:beef:cafe route-map RM-SR-TE-PEER-OUT6 out
+      no neighbor 2002:db8::dead:beef:cafe activate
+   !
+   address-family link-state
+      bgp missing-policy direction in action permit
+      bgp missing-policy direction out action deny
+      neighbor PG-1 activate
+      neighbor PG-1 missing-policy direction in action deny-in-out
+      neighbor PG-1 missing-policy direction out action permit
+      no neighbor PG-2 activate
+      neighbor 192.168.255.1 activate
+      neighbor 192.168.255.1 missing-policy direction in action deny
+      neighbor 192.168.255.1 missing-policy direction out action deny
+      neighbor 192.168.255.2 activate
+      path-selection
+      path-selection role consumer propagator
+   !
+   address-family path-selection
+      bgp additional-paths receive
+      bgp additional-paths send ecmp limit 42
+      neighbor PATH-SELECTION-PG-1 activate
+      neighbor PATH-SELECTION-PG-1 additional-paths receive
+      no neighbor PATH-SELECTION-PG-1 send
```

Comment:
Command is not supported for this level. It exit from address-family path-selection while apply below command. The command should not render